src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,114 @@
+            " |  _| | __| '_ \\ / _` | '_ \\\n" +
+            " | |___| |_| | | | (_| | | | |\n" +
+            " |_____|\\__|_| |_|\\__,_|_| |_|\n" +
+            "\n";
+    public static final String INTRO = " Hi there, I'm Ethan!\n" +
+            " Do you need anything?\n";
+    public static final String BYE = " Bye. See you soon!\n" +
+            "----------------------------BYE----------------------------";
+    public static final String YOU = "----------------------------YOU----------------------------\n";
+    public static final String ETHAN = "---------------------------ETHAN---------------------------\n";
+
+    private static Task[] task_array= new Task[100];
+    private static int taskCount=0;
+
+    //Methods
+    public static void printList(){
+        for(int i=0; i<taskCount; i++) {
+            System.out.println((i+1) + "." + task_array[i]);
+        }
+    }
+
+    public static void addTask(Task t){
+        task_array[taskCount]=t;
+        taskCount++;
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  "+ t);
+        System.out.println("Now you have "+ taskCount+ (taskCount==1?" task":" tasks")+" in the list");
+    }
+
+    public static void runCommand(String user_command) throws DukeException{
+        String[] words= user_command.split(" ");
+        String description;
+        String timing;
+        int dividerPosition;
+        if(user_command.equals("list")){
+            printList();
+        } else if(words[0].equals("done")){
+            int done_index=Integer.parseInt(words[1])-1;
+            task_array[done_index].isDone(true);
+        } else if(words[0].equals("todo")){
+            //Description missing handler
+            try{
+                description=user_command.substring(5);
+                addTask(new Todo(description));
+            } catch(StringIndexOutOfBoundsException e){
+                throw new DukeException("todo incomplete");
+            }
+        } else if(words[0].equals("deadline")){
+            dividerPosition= user_command.indexOf("/");
+            // Divider missing handler
+            if(dividerPosition==-1){
+                throw new DukeException("DL divider missing");
+            }
+            // Deadline incomplete handler
+            try {
+                description = user_command.substring(9, dividerPosition - 1);
+                timing = user_command.substring(dividerPosition + 1);
+                addTask(new Deadline(description, timing));
+            } catch(StringIndexOutOfBoundsException e){
+                throw new DukeException("DL incomplete");
+            }
+        } else if(words[0].equals("event")){
+            dividerPosition= user_command.indexOf("/");
+            addTask(new Event(user_command.substring(6,dividerPosition-1), user_command.substring(dividerPosition+1)));
+        } else{
+            throw new DukeException("invalid");
+        }
+
+    }
```

Comment:
Maybe you can try extracting certain lines of code to make the method shorter? It seems a bit long.